fake_joint_trajectory_controller:
  ros__parameters:
    joints:
      - armlink1_to_servo4
      - base_to_rotary_table
      - left_gripper_joint
      - servo2_to_armlink4
      - servo3_to_armlink3_part1
      - servo5_link_to_armlink1
    write_op_modes:
      - armlink1_to_servo4
      - base_to_rotary_table
      - left_gripper_joint
      - servo2_to_armlink4
      - servo3_to_armlink3_part1
      - servo5_link_to_armlink1